### PR TITLE
Added supports contextPropagators for localActivity.

### PR DESCRIPTION
**Pull request by KovaliovNA**

### CODE REVIEW

**Comment by KovaliovNA** [comment]
@vancexu I have added the test for two cases:

1. Propagators were enabled and we get propagated value in LocalActivity;
2. Propagators were disabled and we get null in LocalActivity from ThreadLocalContext, because the information about current thread was lost.

File: src/test/java/com/uber/cadence/activity/LocalActivityContextPropagationTest.java [hunk 1]
```diff
@@ -0,0 +1,242 @@
+package com.uber.cadence.activity;
```
Comment (vancexu) [inline src/test/java/com/uber/cadence/activity/LocalActivityContextPropagationTest.java:1]: The tests looks good. 
Could you also run `./gradlew licenseFormat` to add license header for this file, before we can merge it?
Thanks

Comment (KovaliovNA) [reply src/test/java/com/uber/cadence/activity/LocalActivityContextPropagationTest.java:18]: License updated
`
Updating license header in src\test\java\com\uber\cadence\activity\LocalActivityContextPropagationTest.java
1 license header(s) updated. A backup of the original file(s) was created in build\tmp\updateLicenseTest\original
`

**Comment by vancexu** [comment]
Thanks again for the contribution.

**Pull request merged**